docs/reference/migration/migrate_3_0.asciidoc

```diff
@@ -91,6 +91,10 @@ characteristics as the former `scan` search type.
 [[breaking_30_rest_api_changes]]
 === REST API changes
 
+==== id values longer than 512 characters are rejected
+
+When specifying an `_id` value longer than 512, the request will be rejected.
```

Comment:
can we be more precise and say 512 UTF-16 codepoints or maybe check the byte length and make it bytes